packages/manager/modules/pci/src/projects/project/storages/databases/add/add.scss

```diff
@@ -1,3 +1,25 @@
 .order-summary {
   z-index: 1009;
 }
+
+tr.table-single-select-row {
+  cursor: pointer;
+}
+tr.table-single-select-row:hover {
+  background-color: #bef1ff;
+}
+.table-selected {
+  background-color: #def8ff;
+}
+.table-selected > td {
+  font-weight: 800 !important;
+}
```

Comment:
```suggestion
.table-selected > td {
  @extend .font-weight-bold;
}
```

Reply:
Same as before, this is not used anymore: https://github.com/ovh/manager/pull/8392/commits/913ab877b1b4954f336b29d90f2d1f0946e00f00.